CMake/cdat_modules/x264_deps.cmake

```diff
@@ -1 +1 @@
-set(X264_DEPS "")
+set(X264_deps ${pkgconfig_pkg})
```

Comment:
thanks @chaosphere2112.care to add a note in the deps file so that someone know it next time?